pavelib/prereqs.py

```diff
@@ -24,6 +24,7 @@
     'requirements/edx/github.txt',
     'requirements/edx/local.txt',
     'requirements/edx/base.txt',
+    'requirements/edx/paver.txt',
```

Comment:
I wasn't sure. It seemed like it might depend upon base, but then I'm not sure why there are others before base. I can move it if folks think it would be clearer.